python/tk_desktop/desktop_window.py

```diff
@@ -17,6 +17,7 @@
 import cPickle as pickle
 import pprint
 import itertools
+import urlparse
 from collections import OrderedDict
```

Comment:
Yeah, desktop is 2.7 only, it runs with the builtin Python, so there's no worry there.